src/client/main.cpp

```diff
@@ -52,6 +52,28 @@ class ProtocachedClient {
     return result;
   }
 
+  ResponseResult Set(const ::std::string &key, const ::std::string &value) {
+    ::protocached::SetRequest request;
+    request.set_key(key);
+    CachedValue cached_value;
+    cached_value.set_value(value);
+    request.mutable_value() -> CopyFrom(cached_value);
+    ::protocached::SetResponse response;
+    ClientContext context;
+
+    Status status = stub_->Set(&context, request, &response);
+
+    ResponseResult result;
+    if (status.ok()) {
```

Comment:
We can probably have a helper method for this, as we need this for both Get/Set and potentially other APIs we have in the future.

Reply:
Refactored.